tests/test_fitsChan.py

```diff
@@ -680,6 +680,25 @@ def test_FitsChanFitsTol(self):
             fv3 = fc2.getFitsS("CTYPE2")
             self.assertEqual(fv3.value, "DEC--TAN")
 
+    def test_FitsChanDM13686(self):
+        """Test that a particular FrameSet will not segfault when
+        we attempt to write it to a FitsChan as FITS-WCS
+        """
+        def readObjectFromShow(path):
```

Comment:
Would this be a useful utility function to lift into astshim itself?

Reply:
Possibly, but I'm reluctant to expand the API without a clear need. I'd have to think more about the name, for one thing.